docs/source/admin/traffic_vault.rst

```diff
@@ -79,6 +79,40 @@ Administration of the PostgreSQL database for Traffic Vault
 
 Similar to administering the Traffic Ops database, the :ref:`admin <database-management>` tool should be used for administering the PostgreSQL Traffic Vault backend.
 
+AES Re-Encryption Tool:
+-----------------------
+
+.. program:: reencrypt
+
+app/db/reencrypt/reencrypt
+=====================================================================
```

Comment:
This heading level (`====`) places this heading above the previous section "AES Re-Encryption Tool:". In HTML terms, this is something like
```html
<section>
  <h1>Traffic Vault Administration</h1>
  <!-- some content... -->
   <section>
    <h2>PostgreSQL</h2>
    <!-- some content... -->
    <section>
      <h3>AES Re-Encryption Tool:</h3>
    </section>
  </section>
  <section>
    <h2>app/db/reencrypt/reencrypt</h2>
    <!-- some content... -->
  </section>
  <!-- some content -->
</section>
```
when I think what you actually want is something more like
```html
<section>
  <h1>Traffic Vault Administration</h1>
  <!-- some content... -->
  <section>
    <h2>PostgreSQL</h2>
    <!-- some content... -->
    <section>
      <h3>AES Re-Encryption Tool:</h3>
	  <section>
		<h4>app/db/reencrypt/reencrypt</h4>
		<!-- some content... -->
	  </section>
    </section>
	<!-- some content... -->
  </section>
  <!-- some content -->
</section>
```
although, the "AES Re-Encryption Tool:" section contains no content besides the content contained in its sub-section in that case, so really you don't need it, so this could just simply be structured:
```html
<section>
  <h1>Traffic Vault Administration</h1>
  <!-- some content... -->
  <section>
    <h2>PostgreSQL</h2>
    <!-- some content... -->
    <section>
	  <h3>app/db/reencrypt/reencrypt</h3>
	  <!-- some content... -->
    </section>
	<!-- some content... -->
  </section>
  <!-- some content -->
</section>
```